<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScrambleDialog</class>
 <widget class="QDialog" name="ScrambleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>任意講座ボタン名称／URL設定</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="opt_title1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="opt_title2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="opt_title3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="opt_title4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>246</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>任意講座ボタン名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="optional1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="optional2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="optional3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="optional4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>246</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>任意講座URL</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ScrambleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ScrambleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
